### ISSUE TITLE
Browse mode: some NVDA commands do not work in this web page

### ISSUE THREAD

**Issue by surfer0627** [nvaccess/nvda#6883]
IN browse mode, some NVDA commands do not work in this web page.
Steps to reproduce:
1. Open link:
http://www.ambassador.com.tw/showtime_list.html
2. Press b to move to "選取" (=select).
3. Press tab several times, NVDA report something like ("10:20, 12:30, 14:40, 16:50".)
Here user could use tab / shift+tab to move accross links, but some NVDA commands are unavailable. It includes:
* Say all
* Find
* Single letter navigation

Also, braille message does not update.

setup:
* browser: Internet Explorer 11.0 / Firefox 51.0
* NVDA: 2016.4

Note:
The content of the web page is in Chinese.

**Comment by feerrenrut** [comment]
Could you please reproduce this with debug level logging enabled and attach the nvda log to this issue? Please read the [Log Files and Crash Dumps](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) wiki page for further instruction on how to do this.

**Comment by surfer0627** [comment]
Hi,
Here is a log file.
Start from line 254.
Thanks.

[nvda.log.txt](https://github.com/nvaccess/nvda/files/772859/nvda.log.txt)

**Comment by surfer0627** [comment]
Perhaps, web site owner do some changes. Now, NVDA commands work again. This issue should be closed.
